### DISCUSSION TITLE
TFT screen capture via serial port

### DISCUSSION THREAD

**Discussion by andimik** [PE5PVB/TEF6686_ESP32#11] [Ideas]
At https://github.com/Bodmer/TFT_eSPI/tree/master/examples/Generic/TFT_Screen_Capture I read it is possible to send a screen capture via serial port using 921600 bauds.

I don't know if it is possible to integrate it with the TEF6686 resp. which tool we need on the PC side.

**Comment by PE5PVB** [comment]
Sorry, it's not possible. We use our own font set and the screen is very dynamic. Why not use a tool like XDRGTK?

**Comment by andimik** [comment]
I see, thanks anyway for the answer.

Sure, xdr-gtk is good enough but I had the idea during the special characters issue, when I had to take a picture of the TEF display. So I wondered if there is a screen capture TOOL.

**Comment by PE5PVB** [comment]
I dare you to write a PC application which simulates it ;-)

**Comment by andimik** [comment]
Let me try with Google Lens ;-)

**Comment by andimik** [comment]
![DSC_5038~3](https://github.com/PE5PVB/TEF6686_ESP32/assets/24510556/c0c654de-6feb-4383-84f9-513aa761a1c4)

(which I could receive during Sporadic E this week)

And Lens found

```RDS 236 kHz

81.90

USB

AUTO BW

MAN

FM

S

1

3

5

M

7

9

+10

+38

20

40

60

80

100

120

MHZ

EQ

37.1

dBuv OFF S/N 26 dB

%

SQ:

PTY: Varied Speech

PS: LA NUEVA

LA NUEVA 87.9 FM

PI: EC72
```

So, this is NOT an option when even the biggest text (frequency) is detect wrong
